### ISSUE TITLE
Remove message "User authentication hooks NOT provided (default user enabled)"

### ISSUE THREAD

**Issue by MarcSkovMadsen** [bokeh/bokeh#13177]
### Problem description

In https://discourse.holoviz.org/t/cannot-execute-script-with-panel/5464 *NoBody* gets confused by the message `User authentication hooks NOT provided (default user enabled)` that is shown with `panel serve`. He thinks this is causing him not to be authenticated to read files on his server.

### Feature description

As authentication is not normally something new users think about I would propose 

- Not showing this message when the server is launched without authentication hooks
- Showing the message `User authentication hooks provided` when the server is started with authentication hooks.

### Potential alternatives

- Keep it
- Improve the message

### Additional information

Phillip Rudiger asked me to report this one with Bokeh in https://github.com/holoviz/panel/issues/5021

**Comment by bryevdv** [comment]
I would support improving the message the message to something like: 

> Bokeh app authentication: default user enabled

and 

> Bokeh app authentication: user authentication hooks provided

for the respective cases.

**Comment by mozart-segfy** [comment]
I have to disagree. It is the exception to have weak authentication, so to keep the warning tone the message should stay as-is. It does not appear to be a confusing message to me.

**Comment by bryevdv** [comment]
> It is the exception to have weak authentication

This is debatable. Auth does not apply to anyone embedding apps into notebooks, for instance. Or anyone just exploring locally to learn about Bokeh apps. Many dashboards are explicitly intended for wide audiences, e.g. are deployed to be publically viewable by anyone, either on a local network or the wide internet. Auth is appropriate when auth is approriate, and even then many users may find it more ergonomic to enforce auth via other existing mechanisms established on their network instead of relying on Bokeh. 

In any case, the message *is* confusing, I have always thought so ever since I wrote it. :) I do intend to update it in some way similar to the manner I described above.
